lib/providers/hlr-lookups-provider.js

```diff
@@ -44,7 +61,7 @@ HLRLookupsProvider.prototype.hlrLookup = function hlrLookup(number) {
     }
 
     if (!isProviderLookupEnabled) {
-      logger.info(logFunc('Returning dummy lookup result because actual calls are prohibited (env ENABLE_HLR_LOOKUPS not set to true)'));
+      logWarningIfRequired(logger, logFunc);
       if (!dummyResponse) {
         dummyResponse = require('./dummy_data/hlr-lookups-provider.json').data;
```

Comment:
I mean to inject dummy data from tests instead of having a condition in main code to fetch dummy data.

I did not checked the whole flow, I assert that `dummyResponse` is a test-related thing?